OTAnalytics/plugin_ui/customtkinter_gui/line_section.py

```diff
@@ -48,9 +106,12 @@ def draw(
         id: str,
         start: tuple[int, int],
         end: tuple[int, int],
-        line_width: int = LINE_WIDTH,
-        line_color: str = LINE_COLOR,
-    ) -> None:
+        is_selected: bool = False,
+        is_edited: bool = False,
+        style_default: dict = DEFAULT_SECTION_STYLE,
+        style_selected: dict = SELECTED_SECTION_STYLE,
+        style_edited: dict = EDITED_SECTION_STYLE,
+    ) -> None:  # sourcery skip: dict-assign-update-to-union
```

Comment:
The caller already decides which style is used by setting the corresponding boolean flags. Instead of different styles and boolean flags as parameters, it is better to only use the style as parameter. The caller (`DummyViewModel`) can than use the appropriate style to draw the lines.